connector_search_engine/models/se_binding.py

```diff
@@ -122,92 +132,64 @@ def _jobify_get_job_description(self, processing_count):
                 model_name=self._name,
             )
 
-    def jobify_recompute_json(self, force_export=False, batch_size=500):
+    def jobify_recompute_json(self, force_export: bool = False):
+        """Create one job per record to recompute the json."""
         # The job creation with tracking is very costly. So disable it.
-        bindings = self.with_context(tracking_disable=True)
-        while bindings:
-            processing, bindings = self._get_binding_to_process(bindings, batch_size)
-            description = self._jobify_get_job_description(len(processing))
-            processing.with_delay(description=description).recompute_json(
-                force_export=force_export
+        for binding in self.with_context(tracking_disable=True):
```

Comment:
I will use the batch method here : _batch.
Computing one by one is not optimal (for example when doing the action on a product with a lot of variant)